### DISCUSSION TITLE
RNGH version 1.10.0 breaks jest.

### DISCUSSION THREAD

**Discussion by ravinhwg** [software-mansion/react-native-gesture-handler#1353] [Q&A] [answered]
I installed RNGH v1.10.0 on a new project and installed react-navigation. In order for tests to work I followed this guide. . Guess what? Looks like the project is fully refactored on version 1.10.0

**Comment by jgonet** [comment, marked as answer]
Please stick to submitting bug reports in the issues section.

This issue was reported here: https://github.com/software-mansion/react-native-gesture-handler/issues/1335
and fixed here: https://github.com/software-mansion/react-native-gesture-handler/issues/1335

The patch will be released in a few days, we need to iron out the rest of the issues coming from the 1.10.0 release.
Sorry for the incovienience.

**Reply by ravinhwg** [reply]
Thank you very much! I had to downgrade to 1.9.0. May I delete this?

**Reply by jgonet** [reply]
I'm not sure if you can delete Discussion / issue but if you can, go for it.

**Reply by ravinhwg** [reply]
Yup! Can't delete. I will do as soon as they allow that. Thank you very much for answering me. I was very confused.
